CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## [Unreleased]
 
+### Behavior Changes
+* Enable log recycling by default.
```

Comment:
And put it at the end.
```suggestion
### Behavior Changes

* Enable log recycling by default.
```

Reply:
@LykxSassinator I updated the comment.

Reply:
How about => 

### Bahavior Changes

* Change format version to 2 from 1 by default.
* Enable log recycling by default.

Reply:
Sure.